Tests/Integration/DotNet/Azure/Storage/Blobs/BlobsStorageBaseFixture.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
```

Comment:
`System` and `Microsoft.Bot.Builder` are not being used.

Reply:
Applied, thanks @ceciliaavila